### ISSUE TITLE
Keras Sequential Model Training Issue

### ISSUE THREAD

**Issue by maran-t** [keras-team/keras#13908]
I used a Keras Sequential Model, **while training my model** the val_accuracy remains constant

```
Epoch 1/10
13/13 [==============================] - 178s 14s/step - loss: 0.5103 - accuracy: 0.8631 - val_loss: 2.7296 - val_accuracy: 0.5676
Epoch 2/10
13/13 [==============================] - 169s 13s/step - loss: 0.2492 - accuracy: 0.9315 - val_loss: 6.5856 - val_accuracy: 0.5676
Epoch 3/10
13/13 [==============================] - 170s 13s/step - loss: 0.2925 - accuracy: 0.9324 - val_loss: 6.6323 - val_accuracy: 0.5676
Epoch 4/10
13/13 [==============================] - 176s 14s/step - loss: 0.3018 - accuracy: 0.9120 - val_loss: 6.6323 - val_accuracy: 0.5676
Epoch 5/10
13/13 [==============================] - 180s 14s/step - loss: 0.1650 - accuracy: 0.9463 - val_loss: 6.6284 - val_accuracy: 0.5676
Epoch 6/10
13/13 [==============================] - 174s 13s/step - loss: 0.1716 - accuracy: 0.9495 - val_loss: 6.6299 - val_accuracy: 0.5676
Epoch 7/10
13/13 [==============================] - 177s 14s/step - loss: 0.1100 - accuracy: 0.9584 - val_loss: 6.6323 - val_accuracy: 0.5676
Epoch 8/10
13/13 [==============================] - 176s 14s/step - loss: 0.1987 - accuracy: 0.9495 - val_loss: 6.6323 - val_accuracy: 0.5676
Epoch 9/10
13/13 [==============================] - 168s 13s/step - loss: 0.1058 - accuracy: 0.9660 - val_loss: 6.6323 - val_accuracy: 0.5676
Epoch 10/10
13/13 [==============================] - 169s 13s/step - loss: 0.1369 - accuracy: 0.9455 - val_loss: 6.6323 - val_accuracy: 0.5676
```

Colab Notebook Link: https://colab.research.google.com/drive/1fwaObStRagmHKZD4eqlHnhbEpC_JT1rx

Thank you!

**Comment by krshashank963** [comment]
Try different learning rate hopefully it will be solved

**Comment by DuyguA** [comment]
There can be several reasons, but #1 reason it happened to me is when:

```
I don't have enough data to train my model
```
i.e. huge network, many layers, many parameters to train but a small training set. Can it be the case for you?

**Comment by Saduf2019** [comment]
@maran-t 

Moving this issue to closed status as there has been no activity, in case you still face the error please create a new issue.
